pkg/syncutil/syncbool_test.go

```diff
@@ -44,4 +44,8 @@ func Test_SyncBool(t *testing.T) {
 	case <-done:
 		// Success
 	}
+
+	if !b.Get() {
+		t.Errorf("channel closed but b.Get() was false")
+	}
```

Comment:
What was happening here?

Reply:
I've rewritten the test now to make it clearer and use existing waits rather than rolling our own.